hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestHoodieMergeOnReadTable.java

```diff
@@ -132,17 +140,27 @@ public void init(HoodieFileFormat baseFileFormat) throws IOException {
 
   @BeforeEach
   public void init() throws IOException {
-    init(HoodieTableConfig.HOODIE_BASE_FILE_FORMAT_PROP.defaultValue());
+    init(HoodieTableConfig.HOODIE_BASE_FILE_FORMAT_PROP.defaultValue(), true);
   }
 
   @AfterEach
   public void clean() throws IOException {
     cleanupResources();
   }
 
-  @Test
-  public void testSimpleInsertAndUpdate() throws Exception {
-    HoodieWriteConfig cfg = getConfig(true);
+  private static Stream<Arguments> populateMetaFieldsParams() {
```

Comment:
This will definitely blow up the runtime if we were to add to every method. I tried to do a decent coverage of operations. We can discuss on how to go about this. Basically this could 2x the runtime of every class where we wish to run it fully against meta disabled and meta enabled. Already, our CI is not able to handle the load. So, atleast once we make CI (move to azure completely) stable with no limits on test runs (as of now, guess there is 1 hour limit in azure, and with our bulk insert support for virtual keys, guess all azure runs are failing), we can revisit this to add more tests with meta disabled.